.travis.yml

```diff
@@ -285,18 +304,24 @@ jobs:
         directories:
           - examples/hello_world_no_reflector/.dart_tool
 
-# Create a merged output build/.
-before_deploy:
-  - pushd examples/hacker_news_pwa
-  - pub run build_runner build -o build --fail-on-severe
-  - popd
-
-deploy:
-  skip_cleanup: true
-  on:
-    repo: dart-lang/angular
-    condition: -d "examples/hacker_news_pwa/.dart_tool/build"
-  provider: firebase
-  message: "Travis deploy from commit $TRAVIS_COMMIT"
-  project: dart-hn
+    # Deploy the PWA application to Firebase.
+    # See https://docs.travis-ci.com/user/build-stages/#Build-Stages-and-Deployments.
+    - stage: deploying
+      script: skip
+      env: PKG="examples/hacker_news_pwa"
+      cache:
+        directories:
+          - examples/hacker_news_pwa/.dart_tool
+      before_deploy:
+        - pushd examples/hacker_news_pwa
+        - pub upgrade
+        - pub run build_runner build -o build --fail-on-severe --config release
+        - popd
+      deploy:
+        skip_cleanup: true
+          on:
```

Comment:
This `on` (and all keys below) are indented too far!

Reply:
See

https://github.com/dart-lang/angular/commit/48552aa6f41c7d16a505d3e28f7c93db02966575
https://travis-ci.org/dart-lang/angular/builds/348726328#L1428